components/sections/Hero.tsx

```diff
@@ -0,0 +1,39 @@
+import { GetHeroSpotData } from "@/lib/GetHeroSpotData";
+
+const Hero = async () => {
+  const heroPlaces = await GetHeroSpotData();
+
+  const heroPlacesList = heroPlaces.map((heroPlace: HeroType) => (
+    <li key={heroPlace.id} className="flex-1">
```

Comment:
**Padding Adjustment and Figma Design Consistency**
Removing the left and right paddings would be beneficial to align the design more closely with our Figma layout. This consistency will ensure a unified look and feel across the application. Please review the relevant sections of the code and make the necessary adjustments.

**Accessibility Implementation:**
Adding accessibility features is crucial for ensuring our application is inclusive and usable by everyone. Could you please integrate appropriate accessibility measures into the codebase? This includes but isn't limited to adding proper alt text for images, labeling form fields, ensuring keyboard navigation, and ARIA attributes where necessary.

For the mobile version background picture please follow the Figma design

![image](https://github.com/NurkaAmre/nomad/assets/107000157/3c4e4cc7-9daf-4a5f-95ba-01786ec3603b)